functions/zoom-uploader.py

```diff
@@ -251,11 +288,21 @@ def create_mediapackage(self):
     def add_tracks(self):
 
         logger.info("Adding tracks")
-        speaker_vids = sorted(
-            self.speaker_videos,
-            key=lambda vid: vid.metadata['track_sequence']
-        )
-        for video in speaker_vids:
+
+        def sorted_videos(vids):
+            return sorted(
+                    vids,
+                    key=lambda vid: vid.metadata['track_sequence']
+                   )
+
+        if self.speaker_videos is not None:
+            videos = sorted_videos(self.speaker_videos)
+        elif self.gallery_videos is not None:
+            videos = sorted_videos(self.gallery_videos)
```

Comment:
are speaker vs gallery videos mutually exclusive? I thought a recording could have both.

Reply:
Since we are currently doing singlestream only (opencast presenter view) we are picking one or the other. But if we know how to do multistream in opencast (do we need to change something with our opencast cluster?) we could include both.

Reply:
Got it. I'll add an issue for me to work out the details of the dual stream workflow.